styles.css

```diff
@@ -1001,3 +987,444 @@ h1 {
+    width: 448px;
+    height: 348px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 24px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .title-post {
+    width: 448px;
+    height: 48px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin-top: 55px;
+  }
+
+  .sup {
+    width: 448px;
+    height: 100px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .tags {
+    margin-top: 0;
+  }
+
+  .card-works {
+    position: absolute;
+    width: 369px;
+    height: 435px;
+    left: 55px;
+    top: 659px;
+    border: 2px solid #091e42;
+    background-image: url(/Images/img2.png);
+    background-position: 100% 100%;
+  }
+
+  .title-post-2,
+  .sup-t,
+  .tags-2 {
+    visibility: hidden;
+  }
+
+  .action {
+    height: 58px;
+    width: 369px;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    top: 377px;
+  }
+
+  .card-works-2 {
+    position: absolute;
+    width: 369px;
+    height: 435px;
+    left: 476px;
+    top: 659px;
+    border: 0 solid #091e42;
+    background-image: url(Images/img3.png);
+  }
+
+  .sup-2 {
+    visibility: visible;
+  }
+
+  .right-block-2 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    position: absolute;
+    width: 369px;
+    height: 257px;
+    left: calc(50% - 369px / 2);
+    bottom: 80px;
+  }
+
+  .action7 {
+    visibility: hidden;
+  }
+
+  .sup12 {
+    position: absolute;
+    top: 0;
+    bottom: 40px;
+  }
+
+  .sup0 {
+    position: absolute;
+    top: 40px;
+  }
+
+  .sup13 {
+    position: absolute;
+    top: 370px;
+  }
+
+  .card-works-3 {
+    position: absolute;
+    left: 900px;
+    right: 0;
+    top: 659px;
+    bottom: 0;
+    border: 0 solid;
+    background-image: url(Images/img4.png);
+  }
+
+  .sup14 {
+    position: absolute;
+    top: 170px;
+  }
+
+  .card-works-4 {
+    position: absolute;
+    width: 369px;
+    height: 435px;
+    left: 476px;
+    top: 1200px;
+    border: 0 solid #091e42;
+    background-image: url(Images/img3.png);
+  }
+
+  .card-works-5 {
+    position: absolute;
+    width: 369px;
+    height: 435px;
+    left: 900px;
+    top: 1200px;
+    border: 0 solid;
+    background-image: url(Images/img4.png);
+  }
+
+  .card-works-6 {
+    position: absolute;
+    width: 369px;
+    height: 435px;
+    left: 55px;
+    top: 1200px;
+    background-image: url(/Images/Mask\ Group.png);
+    background-repeat: no-repeat;
+    background-repeat: 369px 435px;
+    border: 0 solid;
+  }
+
+  .sup23 {
+    position: absolute;
+    top: 370px;
+  }
+
+  .about {
+    position: absolute;
+    width: 100%;
+    height: 1195px;
+    left: 0;
+    top: 2680px;
+    background: #262626;
+    background-image: url(Images/bg7.png);
+    background-repeat: no-repeat;
+    background-position: 100% 20%;
+  }
+
+  .primery {
+    width: 566px;
+    height: 78px;
+    margin-left: 142px;
+    flex: none;
+    order: 0;
+    flex-grow: 0;

```

Comment:
- [ ] kindly check your skills, since the columns are not well aligned as in the FIGMA design.

![image](https://user-images.githubusercontent.com/93630700/208182114-0d8b446f-446c-45f7-a93d-cea878f70167.png)